bostrom/assetlist.json

```diff
@@ -51,6 +51,84 @@
           "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/bostrom/images/hydrogen.svg"
         }
       ]
+    },
+    {
+      "description": "The resource token of Bostrom",
+      "denom_units": [
+        {
+          "denom": "milliampere",
```

Comment:
There must exist a denom_unit with exponent 0, which I assume is milliampere. And then you can have another denom_unit with exponent 3 which you'd use for 'ampere', which is what you've defined as the display denom.
I see the same issue with volt/millivolt (and volt's alias 'A' I think is a typo)

Reply:
Hey @JeremyParish69 ! 

Fixed with  the [3078d12](https://github.com/cosmos/chain-registry/pull/3225/commits/3078d12e6bfa6aaad8974967175f99c65c2fb041)